### ISSUE TITLE
Retain landing page for `write_dwc()`

### ISSUE THREAD

**Issue by peterdesmet** [inbo/camtraptor#308]
https://doi.org/10.1002/rse2.374 states:

> Camtrap DP datasets can be transformed to species occurrence data expressed as Darwin Core Archives (Darwin Core Maintenance Group, [2021](https://zslpublications.onlinelibrary.wiley.com/doi/10.1002/rse2.374#rse2374-bib-0015)), as demonstrated by the write_csv() [sic] function in the “camtraptor” R package (https://inbo.github.io/camtraptor/reference/write_dwc.html; Oldoni et al., [2023](https://zslpublications.onlinelibrary.wiley.com/doi/10.1002/rse2.374#rse2374-bib-0052)).

That function has now moved to camtrapdp (https://inbo.github.io/camtrapdp/reference/write_dwc.html) and we're not planning to export it in camtraptor. We should retain the landing page `https://inbo.github.io/camtraptor/reference/write_dwc.html` however (and maybe redirect to camtrapdp).

**Comment by peterdesmet** [comment]
This should now be solved by https://github.com/inbo/inbo.github.io/commit/5f71ec574b1dd10510cc713018ce189eafc4cab5, but we'll only be able to test the redirect once the `write_dwc()` function is removed from camtraptor.

**Comment by damianooldoni** [comment]
The function `write_dwc()` is now defunct (not just disappearing) via 266ce07 (working in 
#391).

Asking for its documentation via `?write_dwc` returns this:

---

Help on topic 'write_dwc' was found in the following packages:

[Transform a Camera Trap Data Package to a Darwin Core Archive](http://127.0.0.1:8658/help/library/camtrapdp/html/write_dwc.html)
(in package [camtrapdp](http://127.0.0.1:8658/help/library/camtrapdp/html/00Index.html) in library C:/R/library)
[Defunct functions](http://127.0.0.1:8658/help/library/camtraptor/html/defunct.html)
(in package [camtraptor](http://127.0.0.1:8658/help/library/camtraptor/html/00Index.html) in library C:/R/library)

---

Notice that the links copy pasted here will not work. However, they work correctly in RStudio.

At the moment, I still cannot test pkgdown as the vignettes are not totally updated (= errors returning). Staty tuned 📻

**Comment by damianooldoni** [comment]
Added a reference to camtrapdp in documentation via 18e1f40.

**Comment by damianooldoni** [comment]
## Final update
The pkgdown website built locally confirms that:
- In Changelog page the `write_dwc()` is listed among the defunt functions subsection with the right explanation about which function to use instead (and link to it)
- Clicking on `write_dwc()` from the Changelog will redirect to the reference page about Defunct functions. In the text on top has been added the sentence: "`write_dwc()` has been moved to camtrapdp. Check [`camtrapdp::write_dwc()`](https://inbo.github.io/camtrapdp/reference/write_dwc.html) for more information."

**Comment by peterdesmet** [comment]
Locally, the link to camtraptor/reference/write_dwc.html redirects to camtraptor/reference/defunct.html with this content:

<img width="1700" height="1498" alt="Image" src="https://github.com/user-attachments/assets/bd86caa0-1f06-4d80-973c-13cdde428fa0" />

Can we keep this issue open until we have the functional 1.0 website?

**Comment by damianooldoni** [comment]
Thanks @peterdesmet for this. Yes, sure. I will let it open.

**Comment by damianooldoni** [comment]
@peterdesmet: update after releasing camtraptor v1.0. 

The link [`write_dwc()`](https://inbo.github.io/camtraptor/reference/defunct.html) in [CHANGELOG](https://inbo.github.io/camtraptor/news/index.html#deprecated-functionality-1-0-0) works correctly: it points to the [defunct functions](https://inbo.github.io/camtraptor/reference/defunct.html) page. There there is a correct link to [`camtrapdp::write_dwc()`](https://inbo.github.io/camtrapdp/reference/write_dwc.html) in the description.

@peterdesmet: I think this is the behavior you had in mind. Please, close the issue in case you find it correct.

**Comment by peterdesmet** [comment]
Correct, the link (https://inbo.github.io/camtraptor/reference/write_dwc.html) in the article mentioned at the top still works. It now shows:

<img width="821" height="354" alt="Image" src="https://github.com/user-attachments/assets/d2b1a070-4fa7-476c-b0bc-fea8ee3d60dd" />

Which is perfectly fine.
